src/ovp_pipeline/ui/view_models.py

```
@@ -4249,6 +4249,137 @@ def build_atlas_browser_payload(
+                    (pack, cutoff, READER_HOME_RECENT_CRYSTALS_LIMIT),
+                ).fetchall()
+        except sqlite3.DatabaseError:
+            atlas = None
+            recent_rows = []
+    if atlas is None:
+        # Empty placeholder so downstream rendering shows the
+        # ``run ovp-knowledge-index`` hint without special-casing.
+        from ..synthesis.curated_atlas import CuratedAtlas
+        atlas = CuratedAtlas(
+            pack=pack, top_n=READER_HOME_TOP_TOPICS_LIMIT,
+            total_chains=0, entries=tuple(),
```

Ruff (0.15.12)

[warning] Unnecessary `tuple()` call (rewrite as a literal)

Rewrite as a literal

(C408)